src/components/CStreamStatus/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+
+export enum StreamStatus {
+  ONGOING = 'ongoing',
+  PENDING = 'pending',
+  EXPIRED = 'expired',
+}
+
+const streamStatuses = [
+  { label: 'Ongoing', value: StreamStatus.ONGOING },
+  { label: 'Pending', value: StreamStatus.PENDING },
+  { label: 'Expired', value: StreamStatus.EXPIRED },
+];
+
+type CStreamStatusProps = {
+  onChange: (value: StreamStatus) => void;
+};
```

Comment:
enums and typed are related to typescript and should be put together before the streamStatuses array.